pkg/ingester/metrics.go

```diff
@@ -241,6 +244,14 @@ func newIngesterMetrics(
 			return 0
 		}),
 
+		inflightRequestsSummary: promauto.With(r).NewSummary(prometheus.SummaryOpts{
+			Name:       "cortex_ingester_inflight_push_requests_summary",
```

Comment:
Remove `_summary` prefix.

```suggestion
			Name:       "cortex_ingester_inflight_push_requests",
```

Reply:
Note that there's a gauge with that name above.
I previously suggested to add a summary in order to avoid silent breaking change in the metric, as it would be queried the same way but return nonsense.

Reply:
Ok, I see the problem. It's a pity we have to add `_summary` prefix but I don't have a better proposal, so let's move on.